server/topic.go

```diff
@@ -210,6 +210,17 @@ func (t *Topic) runProxy(hub *Hub) {
 		case msg := <-t.broadcast:
 			// Content message intended for broadcasting to recipients
 			log.Printf("t[%s] broadcast %+v", t.name, msg)
+			brdc := &ProxyTopicData{
+				BroadcastReq: &ProxyBroadcast{
+					Id:        msg.id,
+					From:      msg.from,
+					Timestamp: msg.timestamp,
+					SkipSid:   msg.skipSid,
```

Comment:
I think it should be set only when `pub.echo == false`.

Reply:
`Session.publish()` will take care of it:
https://github.com/tinode/chat/blob/devel/server/session.go#L532